pcsx2/gui/Dialogs/SysConfigDialog.cpp

```diff
@@ -202,6 +202,8 @@ Dialogs::SysConfigDialog::SysConfigDialog(wxWindow* parent)
 
 	//NOTE: all pages which are added to SysConfigDialog must be of class BaseApplicableConfigPanel or derived.
 	//		see comment inside UpdateGuiForPreset implementation for more info.
+	AddPage<BiosSelectorPanel>		( pxL("BIOS"),			cfgid.Cpu );
+	AddPage<StandardPathsPanel>		( pxL("Folders"),		cfgid.Paths );
```

Comment:
formatting

Reply:
![image](https://user-images.githubusercontent.com/24227051/129413309-6f7e49c3-d9c8-4a04-befd-9284863cc9a5.png)

How is it displaying it like that?

Reply:
tabs sucking at aligning things maybe, it's not really a huge issue.